pkg/sys/runner/runner.go

```diff
@@ -127,16 +122,14 @@ func (r run) RunContextParseOutput(ctx context.Context, stdoutH, stderrH func(st
 		go parseReader(&wg, stderrP, stderrH)
 	}
 
+	wg.Wait()
 	err = cmd.Wait()
```

Comment:
cmd.Wait() already closes the pipes, hence deferring the pipes closure is not really needed, neither closing them after the cmd.Wait.